### PR TITLE
Minor tweaks to wording

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ navigation_weight: 1
...
-This is a command line tool that can build a service
-worker that will precache the assets for your site and make it work offline.
+This is a command line tool that will build a service
+worker to precache assets for your site and make it work offline.
 
 [Learn More About sw-cli Here](../reference-docs/stable/latest/module-sw-cli.html#main)
 
-If you'd like to make use of the functionality provided by these libraries
-in your own service worker than `sw-lib` would be the best option for you.
+If you'd like to make use of the sw-helpers libraries, and you already have your own
```
Comment (jpmedley) [inline docs/getting-started/index.md]: Change 'like to make use of' to 'like to use'.

Comment (gauntface) [inline docs/getting-started/index.md]: @jpmedley could you make this change and push the PR.

Comment (jpmedley) [inline docs/getting-started/index.md]: This is in Sam's fork. Either he needs to do it, or I'll need to wait for the merge.

**Comment by addyosmani** [comment]
@samdutton Do you think you'll have a chance to address the piece of feedback from Joe in your PR? If not, we can push what we have here and then follow up with a commit to address ourselves. Thanks!

**Comment by gauntface** [comment]
Moving over to a new PR with Sams changes just to get this closed.

**Pull request closed without merging**